### PR TITLE
Separate Attribute filter sub components

### PR DESCRIPTION
**Pull request by ivanmjartan**
JIRA: RAIL-4243

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/8361/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10231/ : FAILURE in 12m 15s

File: libs/sdk-ui-tests/stories/visual-regression/filters/AttributeFilter@next/Components/AttributeFilterItem.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    uri: "some uri",
+};
+
+const AttributeFilterItemExamples = (): JSX.Element => {
+    return (
+        <div style={{ width: 300 }}>
+            <div className="library-component screenshot-target">
+                <h4>AttributeFilterItem unselected</h4>
+                <AttributeFilterItem
+                    item={item}
```
[the remark this answers is not in the document]
Comment (kandl) [reply libs/sdk-ui-tests/stories/visual-regression/filters/AttributeFilter@next/Components/AttributeFilterItem.tsx:34]: (we can keep this interfaces cleanup for followup)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/8364/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10232/ : SUCCESS in 13m 08s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2645

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2409/ : SUCCESS in 1m 07s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2704/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1523/ : SUCCESS in 17m 47s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10233/ : SUCCESS in 15m 18s

**Pull request merged**